Resources/Prototypes/Catalog/Research/technologies.yml

```diff
@@ -213,27 +213,218 @@
   - SurveillanceWirelessCameraMonitorCircuitboard
   - ComputerTelevisionCircuitboard
 
-#- type: technology
-#   name: "ballistic technology"
-#   id: BallisticTechnology
-#   description: Just a fancy term for guns.
-#   icon:
-#     sprite: Objects/Weapons/Guns/Pistols/mk58.rsi
-#     state: icon
-#   requiredPoints: 15000
-#   requiredTechnologies:
-#     - SecurityTechnology
-#
-# - type: technology
-#   name: "direct energy technology"
-#   id: DirectEnergyTechnology
-#   description: Basically laser guns.
-#   icon:
-#     sprite: Objects/Weapons/Guns/Battery/taser.rsi
-#     state: icon
-#   requiredPoints: 15000
-#   requiredTechnologies:
-#     - SecurityTechnology
+- type: technology
+  name: technologies-security-basic
+  id: BasicSecurityTechnology
+  description: technologies-security-basic-description
+  icon:
+    sprite: Objects/Misc/handcuffs.rsi
+    state: handcuffs
+  requiredPoints: 5000
+  requiredTechnologies:
+    - BasicResearch
+  unlockedRecipes:
+    - Handcuffs
+    - Zipties
+    - StunBaton
+    - RiotShield
+    - RiotLaserShield
+    - RiotBulletShield
+    - Flash
+    - FlashPayload
+    - ForensicPad
+    - ForensicScanner
+    - ClothingGlassesEyesSecurity
+    - FlashlightSeclite
+
+- type: technology
+  name: technologies-ballistic-ammunition
+  id: BallisticAmmunitionTechnology
+  description: technologies-ballistic-ammunition-description
+  icon:
+    sprite: Objects/Weapons/Guns/Ammunition/SpeedLoaders/Magnum/magnum_speed_loader.rsi
+    state: base-6
+  requiredPoints: 5000
+  requiredTechnologies:
+    - BasicSecurityTechnology
+  unlockedRecipes:
+    - MagazineBoxPistol
+    - MagazineBoxPistolRubber
+    - MagazineBoxMagnum
+    - MagazineBoxMagnumRubber
+    - MagazineBoxRifle
+    - MagazineBoxRifleRubber
+    - MagazineBoxLightRifle
+    - MagazineBoxLightRifleRubber
+
+- type: technology
+  name: technologies-ballistic-magazines
+  id: BallisticMagazinesTechnology
+  description: technologies-ballistic-magazines-description
+  icon:
+    sprite: Objects/Weapons/Guns/Ammunition/Magazine/Pistol/pistol_mag.rsi
+    state: red-icon
+  requiredPoints: 10000
+  requiredTechnologies:
+    - BallisticAmmunitionTechnology
+  unlockedRecipes:
+    - SpeedLoaderMagnum
+    - SpeedLoaderMagnumHighVelocity
+    - SpeedLoaderMagnumRubber
+    - MagazineRifle
+    - MagazineRifleHighVelocity
+    - MagazineRifleRubber
+    - MagazinePistolSubMachineGunTopMounted
+    - MagazinePistol
+    - MagazinePistolHighVelocity
+    - MagazinePistolRubber
+
+- type: technology
+  name: technologies-ballistic-weaponry
+  id: BallisticWeaponryTechnology
+  description: technologies-ballistic-weaponry-description
+  icon:
+    sprite: Objects/Weapons/Guns/SMGs/wt550.rsi
+    state: icon
+  requiredPoints: 20000
+  requiredTechnologies:
+    - BasicSecurityTechnology
```

Comment:
having this as a prereq isn't necessary because it's already a prereq for `BallisticMagazinesTechnology`